frontend/src/components/Dropdown.tsx

```diff
@@ -2,55 +2,49 @@ import { useState } from "react";
 
 import styles from "./Dropdown.module.css";
 
-export function Dropdown(props: {
+type DropdownProps = {
   options: string[];
-  onSelect: (value: string) => void;
+  onSelect?: (value: string) => void;
   required?: boolean;
-}) {
-  const [selected, setSelected] = useState("Select One");
-  const [isActive, setIsActive] = useState(false);
+};
 
-  const handleOptionClick = (option: string) => {
+export function Dropdown(props: DropdownProps) {
+  const { options, onSelect, required } = props;
+
+  // default value must be "" for required prop to work, https://stackoverflow.com/a/6048891
+  const defaultSelected = "";
+
+  const [selected, setSelected] = useState(defaultSelected);
+
+  function onSelectChange(event: React.FormEvent) {
+    const element = event.target as HTMLSelectElement;
+    const option = element.value;
     setSelected(option);
-    setIsActive(false);
-    props.onSelect(option);
-  };
+    if (onSelect) onSelect(option);
+  }
 
   return (
-    <div
-      className={styles.dropDown}
-      onClick={() => {
-        setIsActive(!isActive);
-      }}
-      role="button"
-      tabIndex={0}
-      onKeyDown={() => {
-        setIsActive(!isActive);
-      }}
+    <select
+      className={`${styles.select} ${selected === defaultSelected ? styles.unselected : ""}`}
+      onChange={onSelectChange}
+      required={required}
```

Comment:
smll thing; can we add a name attribute to this that could be passed in through the props?